packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -63,19 +63,8 @@ export async function fetchSnapshot(
     snapshotDownloader: (url: string, fetchOptions: { [index: string]: any; }) => Promise<IOdspResponse<unknown>>,
 ): Promise<ISnapshotContents> {
     const path = `/trees/${versionId}`;
-    let queryParams: ISnapshotOptions = {};
-
-    if (fetchFullSnapshot) {
```

Comment:
fetchFullSnapshot arg is no longer used, so it should be removed, right?
That said, does it mean that we always fetch full snapshot? Or only tree? What are the default behavior for ODSP?
I think this API is used to fetch only tree, so we should ensure (trough defaults, or through explicit args) it does that.
I'd chose being explicit, i.e. our code not depend on ODSP defaults and always clearly communicating what it wants (i.e. setting all these queryParams)